app/assets/javascripts/discourse/app/components/topic-admin-menu.gjs

```diff
@@ -0,0 +1,310 @@
+                        class="popup-menu-btn"
+                      />
+                    </li>
+                  {{/if}}
+                </ul>
+
+                <ul class="topic-admin-menu-time">
+                  {{#if this.currentUser.canManageTopic}}
+                    <li class="admin-topic-timer-update">
+                      <DButton
+                        @label="topic.actions.timed_update"
+                        @action={{fn this.onButtonAction "showTopicTimerModal"}}
+                        @icon="far-clock"
+                        class="popup-menu-btn"
+                      />
+                    </li>
+
+                    {{#if this.currentUser.staff}}
+                      <li class="topic-admin-change-timestamp">
+                        <DButton
+                          @label="topic.change_timestamp.title"
+                          @action={{fn
+                            this.onButtonAction
+                            "showChangeTimestamp"
+                          }}
+                          @icon="calendar-alt"
+                          class="popup-menu-btn"
+                        />
+                      </li>
+                    {{/if}}
+
+                    <li class="topic-admin-reset-bump-date">
+                      <DButton
+                        @label="topic.actions.reset_bump_date"
+                        @action={{fn this.onButtonAction "resetBumpDate"}}
+                        @icon="anchor"
+                        class="popup-menu-btn"
+                      />
+                    </li>
+
+                    <li class="topic-admin-slow-mode">
+                      <DButton
+                        @label="topic.actions.slow_mode"
+                        @action={{fn
+                          this.onButtonAction
+                          "showTopicSlowModeUpdate"
+                        }}
+                        @icon="hourglass-start"
+                        class="popup-menu-btn"
+                      />
+                    </li>
+                  {{/if}}
+                </ul>
+
+                <ul class="topic-admin-menu-undefined">
+                  {{#if this.currentUser.staff}}
+                    <li class="topic-admin-moderation-history">
+                      <DButton
+                        @label="review.moderation_history"
+                        @href={{this.topicModerationHistoryUrl}}
+                        @icon="list"
+                        class="popup-menu-btn"
+                      />
+                    </li>
+                  {{/if}}
+
+                  {{#each this.extraButtons as |button|}}
+                    <li>
+                      <DButton
+                        @label={{button.label}}
+                        @translatedLabel={{button.translatedLabel}}
+                        @icon={{button.icon}}
+                        class={{concatClass "popup-menu-btn" button.className}}
```

Comment:
do we still need to use "popup-menu-btn" ? what style do we use on the post menu?